processor/src/test/java/org/mapstruct/ap/test/decorator/PersonMapper.java

```diff
@@ -33,4 +34,6 @@
     PersonDto personToPersonDto(Person person);
 
     AddressDto addressToAddressDto(Address address);
+
+    void updateAddressFromDto(AddressDto dto, @MappingTarget Address address);
```

Comment:
what about the combi: `Address updateAddressFromDto(AddressDto dto, @MappingTarget Address address);`.. Should we have a UT for that as well?